### PR TITLE
SFR-537 Add LCCN endpoint to CCE API

### PR DESCRIPTION
**Pull request by mwbenowitz**
This adds a LCCN search endpoint to the API for this project and includes any LCCNs that are attached to CCE records in the API responses. These values are queried in the ElasticSearch index and fetched from the database along with the rest of the data when the response fields are gathered.

### CODE REVIEW

File: api/prints/search.py [hunk 1]
```diff
@@ -77,6 +77,31 @@ def regQuery(regnum):
...
+@search.route('/lccn/<lccn>', methods=['GET'])
+def lccnQuery(lccn):
+    page, perPage = MultiResponse.parsePaging(request.args)
+    sourceReturn = request.args.get('source', False)
+    matchingDocs = elastic.query_lccn(lccn, page=page, perPage=perPage)
+    regResponse = MultiResponse(
+        'number',
+        matchingDocs.hits.total,
+        request.base_url,
+        lccn,
```
Comment (seanredmond) [inline api/prints/search.py]: Remind me, are the LCCNs normalized in the db or ES? Or do you have to get this param just right to match?

Comment (mwbenowitz) [reply api/prints/search.py]: The LCCNs are normalized using your `lccnorm` package before being stored/indexed. So you need to pass a properly formatted LCCN here to find a match

Comment (seanredmond) [reply api/prints/search.py]: It would be great to normalize the search term, too. You'd have to figure out what to do if wouldn't normalize

Comment (mwbenowitz) [reply api/prints/search.py]: No problem I can validate and normalize before querying ElasticSearch, if that fails I think we'd return a plain 400 error as an unparseable lccn would qualify as a bad request

**Comment by mwbenowitz** [comment]
This has been merged but a Github error has mangled this PR, if you look at the `development` branch you can see the merged code. Leaving this comment for future reference, this PR should be considered `Merged` and not `Closed`

**Pull request closed without merging**